lib/subscriptions_test_kit/common/subscription_conformance_verification.rb

```diff
@@ -66,7 +66,6 @@ def subscription_verification(subscription_resource)
                             profile_url: 'http://hl7.org/fhir/uv/subscriptions-backport/StructureDefinition/backport-subscription')
 
       cross_version_extension_check(subscription)
-      subscription
```

Comment:
@karlnaden The only difference between the `subscription` that was being returned and the `subscription_resource` being passed in to this method is a `JSON.parse`. Since we now have to do a `JSON.parse` in the caller anyway, I removed this return statement. I also found it a little confusing the way it was, because I had to be very careful to check that this method isn't making other modifications to the subscription. I don't think it is, but please double check me on this.

Reply:
I agree - no changes made in here.